backend/de.metas.business/src/main/java/de/metas/invoice/interceptor/C_InvoiceLine.java

```diff
@@ -99,13 +109,9 @@ public void copyDimensionFromHeader(final I_C_InvoiceLine invoiceLine)
 	{
 		final org.compiere.model.I_C_Invoice invoice = invoiceLine.getC_Invoice();
 		invoiceLine.setM_SectionCode_ID(invoice.getM_SectionCode_ID());
```

Comment:
is this line still needed?

Reply:
I think it is. Added the same docu as for the order lines: 

```only update the section code and user elements. It's not specified if the other dimensions should be inherited from the invoice header to the lines
```